traitquestdev/footer.php

```diff
@@ -0,0 +1,28 @@
+<footer class="footer">
+	<div id="traitquestInfo" class="col-md-6">
```

Comment:
i was thinking to add the details for index page, other pages will not have all the info as well as the contact us form to keep it simplified.